docs/release/release_0_5_6.md

```diff
@@ -125,6 +125,8 @@ Read on for the full list of changes since 0.5.5.
 - add documentation for new path tool ([docs#547](https://github.com/napari/docs/pull/547))
 - Add 0.5.6 release notes ([docs#548](https://github.com/napari/docs/pull/548))
 - 0.5.6 relnotes updates ([docs#552](https://github.com/napari/docs/pull/552))
+- Add napari/napari#7538 to release notes ([docs#554](https://github.com/napari/docs/pull/554)
```

Comment:
As there are 3PR now, maybe update name of this PR?

Reply:
done

Reply:
wait but I have to add the change here LOL
```suggestion
- Add missing PRs to release notes ([docs#554](https://github.com/napari/docs/pull/554)
```